kustomize/go.mod

```diff
@@ -6,6 +6,12 @@ require (
 	github.com/pkg/errors v0.8.1
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
+	gopkg.in/inf.v0 v0.9.1 // indirect
 	sigs.k8s.io/kustomize/v3 v3.2.0
 	sigs.k8s.io/yaml v1.1.0
 )
+
+// Temporarily drop this replacement, and select a specific
+// version of sig.k8s.io/kustomize (the API module) before
+// tagging and releasing this module (the kustomize CLI).
```

Comment:
Maybe add this comment into releasing/README.md as well.

Reply:
will do, once we practice it with the next release.
probably best to do it as part of a script, since its easy to make a mistake.

Reply:
Sounds good, a script is even better.